site/src/ThemeSelector.jsx

```diff
@@ -135,69 +148,76 @@ const ThemeSelector = ({
     const themes = [...new Set(layers.map((layer) => layer.theme))];
 
     return (
-      <Box p={1} className>
-        {themes.map((theme) => {
-          const types = filterUniqueByType(
-            layers.filter((layer) => layer.theme === theme)
-          );
-
-          const children = types.map((t) => selectedTypes[t.type]);
-
-          return (
-            <Grid container width={200}>
-              <Grid
-                className={
-                  theme === "divisions" ? "tour-layers-checkboxes" : ""
-                }
-                item
-                xs={10}
-              >
-                <div>
-                  <FormControlLabel
-                    label={format(theme)}
-                    className="theme-selector-checkbox"
-                    sx={{
-                      height: "16px",
-                    }}
-                    control={
-                      <Checkbox
-                        size="small"
-                        checked={
-                          selectedThemes[theme] && children.includes(true)
-                        }
-                        indeterminate={
-                          children.includes(true) && children.includes(false)
-                        }
-                        onChange={() => handleThemeChange(theme)}
-                      />
-                    }
-                  />
-                </div>
-                {types.length > 1 && (
-                  <Box sx={{ display: "flex", flexDirection: "column", ml: 3 }}>
-                    {types.map((layer) => (
-                      <FormControlLabel
-                        label={format(layer.type)}
-                        className="theme-selector-checkbox"
-                        control={
-                          <Checkbox
-                            size="small"
-                            checked={selectedTypes[layer.type]}
-                            onChange={() => handleTypeChange(layer.type)}
-                          />
-                        }
-                      />
-                    ))}
-                  </Box>
-                )}
-              </Grid>
-              <Grid item xs={2}>
-                {renderPinThemeIcon(theme)}
+      <ThemeProvider theme={muiTheme}>
```

Comment:
This theme statement is what replaces the typography to use the same font styles as the rest of the site.

Reply:
Should we wrap the entire app in this theme provider? I'm planning on updating more components around the site to use Material.